packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -196,9 +199,11 @@ def __call__(self, message: str) -> None:
 class SocketIONodeProgressCompleteWaiter:
     node_id: str
     logger: logging.Logger
+    product_url: AnyUrl
     _current_progress: dict[NodeProgressType, float] = field(
         default_factory=defaultdict
     )
+    _last_poll_timestamp: datetime = datetime.now(tz=UTC)  # noqa: RUF009
```

Comment:
we tend to use `arrow.utcnow.datetime` so there is no risk of having local time with whatever wrong timezone. also here how you initialize it will not work well.
`field(default_dict=lambda: datetime.now(tz=UTC))` is the correct way I believe